redisson/src/main/java/org/redisson/api/RSortableAsync.java

```diff
@@ -153,5 +163,127 @@
      * @return length of sorted data
      */
     RFuture<Integer> sortToAsync(String destName, String byPattern, List<String> getPatterns, SortOrder order, int offset, int count);
-    
+
+	/**
+	 * Parameters that redis sort command supported.
+	 *
+	 * @author PepeLu
+	 */
+	public class RSortParameters {
```

Comment:
Please conform to common API design and avoid this object

Reply:
Done.